plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyJob.java

```diff
@@ -47,27 +59,31 @@
     private Binding binding;
     private String name;
     private Thread t;
+    
 
-    public GroovyJob(String name, String script){
+    public GroovyJob(String name, String script, String groovyInterpreter){
```

Comment:
I don't under why you set a String argument to define the groovy engine.
I think you can remove it and use the original code. No ?